src/components/pages/not-found.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+    
+    <div class="page-not-found__wrapper">
+ 
+        <div class="page-not-found__404">
+            404
+        </div>
+
+        <div class="page-not-found__text">
+            {{ trad.message404[lang] }}
+        </div>
+
+        <router-link :to="{ name: 'home', params: { lang: lang }}"
+            class="page-not-found__get-back-link">
+            {{ trad.backHome[lang] }}
+        </router-link>
+
+    </div>
+    
+</template>
+
+<script>
+import { setMetas } from '../../utils';
+
+//- Traductions
+import trad404 from '../../traductions/not-found';
+
+export default {
+    name: 'not-found',
+    data: function () {
+        return {
+            trad: trad404
+        }
+    },
+    props: {
+        lang: String
+    },
+    methods: {
+        setPageNotFoundMetas: function () {
+            setMetas({
+                title:         this.trad.metas.title[this.lang],
+                description:   this.trad.metas.description[this.lang],
+                lang:          this.lang
+            });
+        }
+    },
+    beforeRouteEnter (to, from, next) {
+        next((vm) => {
+            vm.setPageNotFoundMetas();
```

Comment:
It sounds a bit overkill to declare `setPageNotFoundMetas` as a component public method since it is just used in this place and is not intended to be called from outside